includes/class-wc-payments-checkout.php

```diff
@@ -74,6 +74,8 @@ public function __construct(
 		$this->customer_service = $customer_service;
 
 		add_action( 'wc_payments_add_payment_fields', [ $this, 'payment_fields' ] );
+		add_action( 'woocommerce_order_action_capture_charge', [ $this->gateway, 'capture_charge' ] );
+		add_action( 'woocommerce_order_action_cancel_authorization', [ $this->gateway, 'cancel_authorization' ] );
```

Comment:
Totally agree that this isn't really a hook associated with checkout and doesn't really merit a mention in this class. In previous attempts to refactor these hooks, the `WC_Payments_UPE_Checkout` and `WC_Payments_Checkout` classes have been used as catch-all holders for payment gateway hooks, since--unlike the gateway constructors--we could be confidently assured that thsee checkout classes would only be initialised once.

However, your refactor in #6326 seems like a much more coherent and concrete solution to this problem. I would probably support the change here in #6313 as a temporary solution--in my mind preferable to retaining this in the class constructor, attaching redundant hooks, and having those empty methods fire unnecessarily from the Split UPE gateways; but if you were able to move this to the [`WC_Payment_Gateway_WCPay::init_hooks()` function](https://github.com/Automattic/woocommerce-payments/blob/add/init-hooks/includes/class-wc-payment-gateway-wcpay.php#L419) in #6326, I would readily support that as a much more appropriate long-term solution and would gladly invest all hopes and faith in that blessed PR instead. 🙏